Nodejs/Product/Nodejs/Snippets/1033/Nodejs/http_delete.snippet

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>delete</Title>
+      <Description>Code Snippet for handling an HTTP DELETE request with Express</Description>
+      <Shortcut>http_delete</Shortcut>

```

Comment:
This actually looks like express delete, not http.delete (which doesn't exist anyways) - same with some of the other snippets.